src/linter/rules/__tests__/invalid-change-version.test.mjs

```diff
@@ -70,17 +67,30 @@ changes:
 
     invalidChangeVersion(context);
 
-    strictEqual(context.report.mock.callCount(), 1);
+    strictEqual(context.report.mock.callCount(), 2);
 
-    const call = context.report.mock.calls[0];
+    const first_call = context.report.mock.calls[0];
 
-    deepStrictEqual(call.arguments, [
+    deepStrictEqual(first_call.arguments, [
+      {
+        level: 'error',
+        message: 'Missing version field in the API doc entry',
+        position: {
+          start: { line: 3 },
+          end: { line: 3 },
+        },
+      },
+    ]);
+
+    const second_call = context.report.mock.calls[1];
+
+    deepStrictEqual(second_call.arguments, [
       {
         level: 'error',
         message: 'Missing version field in the API doc entry',
         position: {
-          start: { line: 1, column: 1, offset: 1 },
-          end: { line: 1, column: 1, offset: 1 },
+          start: { line: 4 },
+          end: { line: 4 },
         },
       },
```

Comment:
Can we use partialDeepStrictEqual / deepStrictEqual for both calls in one assertion?